csharp/Platform.Data.Doublets/Memory/Split/Generic/SplitMemoryLinksBase.cs

```diff
@@ -833,24 +835,25 @@ public virtual TLink Create(IList<TLink> restrictions)
                 _dataMemory.UsedCapacity += LinkDataPartSizeInBytes;
                 _indexMemory.UsedCapacity += LinkIndexPartSizeInBytes;
             }
-            return freeLink;
+            return handler(null, GetLinkStruct(freeLink));
```

Comment:
May be 
```suggestion
            return handler(Link<TLink>.Null, GetLinkStruct(freeLink));
```

Reply:
Yes, we can make like this for simplicity.